src/simulation/openables/BronzeHAMChest.ts

```diff
@@ -0,0 +1,27 @@
+import LootTable from '../../structures/LootTable';
+import SimpleOpenable from '../../structures/SimpleOpenable';
+import { rand } from '../../util';
+
+const rolls = rand(1, 5);
```

Comment:
this is not doing what you think its doing

to get random rolls every time you need to make a table with each possible outcome and then roll that table